Update category summaries (#33)
scientific-python/tools.scientific-python.org: content/_index.md

title: Tools & Services
---

# Community Tools & Services

We maintain a collection of tools that help projects across the ecosystem.
We also offer a small selection of hosted services.

{{< grid columns="1 1 1 5" >}}

{{< card >}}
title = 'For Web'
link = 'https://theme.scientific-python.org'
body = '''
A [Hugo](https://gohugo.io/) theme used by [NumPy](https://numpy.org/), [SciPy](https://scipy.org/),
[Scientific Python](https://scientific-python.org/), and other sites.
'''
{{< /card >}}

{{< card >}}
title = 'For Development'
link = '/utilities'
body = '''
Python developer tools used by packages across the ecosystem.
'''
{{< /card >}}

{{< card >}}
title = 'For Organization'
link = '/community'
body = '''

> Calender, forum, and other services for organization the community.

Reviewer: ```suggestion
Calendar, forum, and other services for organizing the community.
```

'''
{{< /card >}}

{{< card >}}
title = 'For Insight'
link = '/analytics'
body = '''

> Through analytics, better understand project development and community health.

Reviewer: ```suggestion
Through analytics, to better understand project development and community health.
```

'''
{{< /card >}}

{{< card >}}
title = 'For GitHub'
link = '/actions'
body = '''

> Actions for automating various aspects of project development.

Reviewer: ```suggestion
GitHub Actions for automating various aspects of project development.
```
